src/ot_orchestration/dags/config/genetics_etl.yaml

```diff
@@ -1,21 +1,78 @@
-- id: "ot_gene_index"
-- id: "ot_variant_index"
-- id: "ot_variant_to_gene"
-  prerequisites:
-    - "ot_variant_index"
-    - "ot_gene_index"
-- id: "ot_colocalisation_ecaviar"
-- id: "ot_colocalisation_coloc"
-- id: "ot_locus_to_gene_train"
-  prerequisites:
-    - "ot_variant_index"
-    - "ot_variant_to_gene"
-    - "ot_colocalisation_ecaviar"
-    - "ot_colocalisation_coloc"
-- id: "ot_locus_to_gene_predict"
-  prerequisites:
-    - "ot_locus_to_gene_train"
-    - "ot_variant_index"
-    - "ot_variant_to_gene"
-    - "ot_colocalisation_ecaviar"
-    - "ot_colocalisation_coloc"
+cluster_name: otg-etl
+gwas_catalog_manifests_path: gs://gwas_catalog_data/manifests
+l2g_gold_standard_path: gs://genetics_etl_python_playground/input/l2g/gold_standard/curation.json
+release_dir: gs://genetics_etl_python_playground/releases/24.09
+
+steps:
+  - id: study_validation
+    params:
+      study_index_path:
+        - gs://gwas_catalog_data/study_index
+        - gs://eqtl_catalogue_data/study_index
+        - gs://ukb_ppp_eur_data/study_index
+        - gs://finngen_data/r11/study_index
+      target_index_path: gs://genetics_etl_python_playground/releases/24.06/gene_index
```

Comment:
It's very likely we will deprecate Gentropy's gene index, and stick with the target index from the Platform. Is it possible to use the one from the Platform.
Isn't there a circular dependency problem here? The ETL needs Gentropy ETL outputs, but to distinguish Gentropy valid and invalids we should use the Platform ETL outputs.

Reply:
As we discussed. The Genetics ETL will be cut into pieces and mixed within the Platform ETL. This way we will have no trouble when running different steps that have in-dag dependencies between processes.